private_sharing/models.py

```diff
@@ -631,6 +632,15 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class ProjectOntology(models.Model):
```

Comment:
Why isn't this a new ManyToManyField on the project model itself?

(Also, I don't see how this model connects to a project as it's currently written...)